src/libs/linkify-urls-in-code.js

```diff
@@ -24,6 +24,9 @@ export const editTextNodes = (fn, el) => {
 		return;
 	}
 	for (const textNode of getTextNodes(el)) {
+		if (textNode.textContent.length < 11) { // Shortest url: http://j.mp
```

Comment:
Realistically we can even increase the number to skip already-short URLs like that.